source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -34,27 +34,22 @@ If __Switch to Production__ is at the bottom of the left-hand menu, toggle it on
 To set up your profile, you need to copy credentials from your Worldpay account
 into your GOV.UK Pay account.
 
-You will need both your [Worldpay
+You must separately sign in to both your [Worldpay
 account](https://secure.worldpay.com/sso/public/auth/login.html) and your
-[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login) open
-and signed in, in different web browser tabs.
+[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login). You
+should have both accounts open, in different web browser tabs.
 
-1. Select __Profile__ in the left-hand menu in your Worldpay account.
+1. In your Worldpay browser tab, select __Profile__ in the left-hand menu. If you
+are prompted to choose a merchant code, select the appropriate one.
 
 1. In your GOV.UK Pay browser tab, go to __My services__ to select the live
 service you want to set up. Select __Settings__, then __Account credentials__,
 then __Edit credentials__.
 
-1. In your Worldpay browser tab, select __Account__ in the left-hand
-menu.
-
-1. From the __Identification__ section, copy the Worldpay __Merchant Code__ into
+1. In Worldpay, copy the Worldpay __Merchant Code__ from __Identification__ into
 the GOV.UK Pay __Merchant ID__ field.
```

Comment:
we're just about to change this to Merchant Code in GOV.UK Pay to make it less confusing, sorry, should've mentioned before!

Reply:
i'll ping you when i sign this off, then it can be changed then so it doesn't hold this up

Reply:
another thing for a future mini PR, is to say when you test it using a payment link, to open that up in another window because it's not particularly easy to navigate back to your admin site account once you've gone on the payment link journey

Reply:
Sure, not sure where to put that info but happy if you want to put it in a Trello card or just email me/add a comment here and I can open a new PR?

Reply:
> we're just about to change this to Merchant Code in GOV.UK Pay to make it less confusing, sorry, should've mentioned before!

No problem, just ping me as you said